packages/react-components/react-accordion/src/stories/Accordion.stories.tsx

```diff
@@ -25,10 +25,6 @@ export default {
     },
   },
   argTypes: {
-    inline: {
```

Comment:
- These props are not for `Accordion` 
- They are displayed without JSDoc
   ![image](https://user-images.githubusercontent.com/14183168/170253752-a501727b-bb9f-4753-b46f-54a61e0e1522.png)